README.rst

```diff
@@ -33,23 +33,38 @@
         :target: https://pypi.org/project/diffpy.fourigui/
 
 .. |Tracking| image:: https://img.shields.io/badge/issue_tracking-github-blue
-        :target: https://github.com/diffpy/diffpy.fourigui/issues
+        :target: https://github.com/diffpy/fourigui/issues
 
-Tool for visualizing 3D diffraction and PDF Images.
+Tool for visualizing 3D diffraction and PDF Images
 
-* LONGER DESCRIPTION HERE
+Fourigui is an interactive visualization tool with an embedded Fourier transformation functionality, designed to
+process 3D reciprocal space scattering volumes to 3D atomic pair distribution functions (3D PDFs).
 
-For more information about the diffpy.fourigui library, please consult our `online documentation <https://diffpy.github.io/diffpy.fourigui>`_.
+Fourigui is a tool to visualize and process 3D data sets written with the python programming language. Fourigui always
+displays one slice perpendicular to one axis and allows scrolling through the 3D data set along the given axis with a
+slider. It shows feedback values such as global and local maxima, minima or NAN ratios. The matplotlib panel e.g. for
+zooming and saving figures is featured. Fourigui is designed for the processing of 3D atomic pair distribution
+functions (PDFs). One can load a 3D reciprocal space scattering volume which can be Fourier transformed to the 3D PDF.
+Thereby, one can apply cut off frequencies beyond and below given Q values, compare the results and switch between the
+scattering volume in reciprocal space and 3D PDF in real space.
+
+
+For more information about the diffpy.fourigui library, please consult our `online documentation <https://diffpy.github.io/fourigui>`_.
 
 Citation
 --------
-
 If you use diffpy.fourigui in a scientific publication, we would like you to cite this package as
 
-        diffpy.fourigui Package, https://github.com/diffpy/diffpy.fourigui
+        diffpy.fourigui Package, https://github.com/diffpy/fourigui
 
 Installation
 ------------
+The diffpy.fourigui package requires Python 3.10 or later and
+the following software:
+
+* ``matplotlib`` - graphing tool
```

Comment:
this is not needed.  Please conform to the cookiecutter template, and only update it when there is missing information that is in the original fourigui docs.